packages/cli/src/editor/helpers/document-title.ts

```diff
@@ -6,12 +6,22 @@ let unsavedProps = false;
 let tabInactive = false;
 let renderJobs: RenderJob[] = [];
 
+declare global {
```

Comment:
All global variables are defined in core/src/index.ts.
Can you move it there as well and update the variable name with a `remotion_` prefix?